src/thor/worker.cc

```diff
@@ -100,6 +100,7 @@ thor_worker_t::thor_worker_t(const boost::property_tree::ptree& config,
 
   max_timedep_distance =
       config.get<float>("service_limits.max_timedep_distance", kDefaultMaxTimeDependentDistance);
+  max_timedep_dist_matrix = config.get<float>("service_limits.max_timedep_distance_matrix", 0.0f);
```

Comment:
I always make those floats as well, and every time I wonder why not a `uint23_t` :)